Quotient/events/roommessageevent.cpp

```diff
@@ -215,7 +256,7 @@ QString RoomMessageEvent::replacedEvent() const
     if (!content() || !hasTextContent())
         return {};
 
-    return isReplacement(_relatesTo) ? _relatesTo->eventId : QString();
+    return isReplacement(relatesTo()) ? relatesTo()->eventId : QString();
```

Comment:
Avoid deserialising the same structure twice.
```suggestion
    const auto er = relatesTo();
    return isReplacement(er) ? er->eventId : QString();
```